Formula/s/spice-server.rb

```diff
@@ -0,0 +1,51 @@
+class SpiceServer < Formula
+  desc "Implements the server side of the SPICE protocol"
+  homepage "https://www.spice-space.org/"
+  url "https://gitlab.freedesktop.org/-/project/62/uploads/54a0f9f5d1840e1ad8060cb560f3dde6/spice-0.16.0.tar.bz2"
+  sha256 "0a6ec9528f05371261bbb2d46ff35e7b5c45ff89bb975a99af95a5f20ff4717d"
+  license "LGPL-2.1-or-later"
+  head "https://gitlab.freedesktop.org/spice/spice.git", branch: "master"
+
+  depends_on "spice-protocol" => [:build, :test]
+  depends_on "pkgconf" => :test
+
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "jpeg"
+  depends_on "lz4"
+  depends_on "openssl@3"
+  depends_on "opus"
+  depends_on "orc"
+  depends_on "pixman"
+
+  uses_from_macos "cyrus-sasl"
+  uses_from_macos "zlib"
+
```

Comment:
systemd is indirect dependency
```suggestion

  on_linux do
    depends_on "systemd"
  end
```

Reply:
Updated.